BraintreeDropIn.podspec

```diff
@@ -27,11 +27,11 @@ Pod::Spec.new do |s|
     s.source_files  = "BraintreeDropIn/**/*.{h,m}"
     s.public_header_files = "BraintreeDropIn/Public/*.h"
     s.frameworks = "UIKit"
-    s.dependency "Braintree/Card", "~> 4.23"
-    s.dependency "Braintree/Core", "~> 4.23"
-    s.dependency "Braintree/UnionPay", "~> 4.23"
-    s.dependency "Braintree/PaymentFlow", "~> 4.23"
-    s.dependency "Braintree/PayPal", "~> 4.23"
+    s.dependency "Braintree/Card", "~> 4.25"
+    s.dependency "Braintree/Core", "~> 4.25"
+    s.dependency "Braintree/UnionPay", "~> 4.25"
+    s.dependency "Braintree/PaymentFlow", "~> 4.25"
+    s.dependency "Braintree/PayPal", "~> 4.25"
```

Comment:
These dependencies should all be bumped to at least `4.26.1` since anything earlier will have App Store submission issues.

Reply:
It looks like `pod update` was run, which will update the dependencies (and Podfile.lock) for the Demo app only. This file needs to be changed manually to specify that we require a minimum of `4.26.1`.